docs/06-concepts/11-authentication/04-providers/06-custom-providers.md

```diff
@@ -113,25 +113,53 @@ class MyAuthenticationEndpoint extends Endpoint {
 
 The example above shows how to create an `AuthenticationResponse` with the auth token and user information.
```

Comment:
Suggestion: Please check the header structure on this page.
Currently it is
- Server setup
- Managing Authentication Keys and Signing Out Users
- Client Setup

These are different abstraction levels. Either we can formulate this page as a guide to set it up or we can explain the different steps.

Reply:
fixed it